docs: added missing API rate documentation (#1607)
spectrocloud/librarium: docs/api-content/api-docs/1-introduction.md

 --header 'Accept: application/json' \
 --header 'ProjectUid: yourProjectUid' \
 --header 'apiKey: yourAPIKey'
```

## Rate Limits

The API rate limits are as follows:

<br />
* There is a limit of ten API requests per second for each source IP address. The API supports additional bursts through the usage of a burst queue. The default burst queue size is set to five. You could make 50 (10 * 5) requests in seconds before the API returns a `429 - TooManyRequests` error. Refer to the [Endpoint Prefix Rate](#endpointprefixrate) for additional information.

* The API request limits are categorized by the parent resources, such as `/v1/cloudconfig/:uid` and `/v1/roles`. You can find a list of all resource types in the [API documentation](v1/palette-apis-4-0.info.mdx). The requests are counted together if you make multiple requests to the same resource type but use different sub-resources. For example, if you make five requests to `/v1/clusterprofiles` and five requests to `/v1/clusterprofiles/macros`, the requests are counted together as ten requests to the resource `clusterprofiles`.

> * In case of too many requests, the user will receive an error with HTTP code `429` - `TooManyRequests.` In that event, we recommend retrying the API call after a few moments. 

Reviewer: ```suggestion
* If too many requests are issued, you may receive an error with HTTP code `429` - `TooManyRequests.` We recommend retrying the API call after a few moments. 
```

## Endpoint Prefix Rate

| **Endpoint Prefix** | **Request Per Second** | **Burst Size** | **Max with Burst** |
|-----------------|--------------------|------------|--------------|
| /v1/auth | 10 | 5 | 50 |
| /v1/nats | 10 | 5 | 50 |
| /v1/users | 10 | 5 | 50 |
| /v1/userprofiles | 10 | 5 | 50 |
| /v1/permissions | 10 | 5 | 50 |
| /v1/roles | 10 | 5 | 50 |
| /v1/teams | 10 | 5 | 50 |
| /v1/tenants | 10 | 5 | 50 |
| /v1/projects | 10 | 5 | 50 |
| /v1/plans | 10 | 5 | 50 |
| /v1/payments | 10 | 5 | 50 |
| /v1/audits | 10 | 5 | 50 |
| /v1/apiKeys | 10 | 5 | 50 |
| /v1/datasinks | 10 | 5 | 50 |
| /v1/notifications | 10 | 5 | 50 |
| /v1/events | 10 | 5 | 50 |
| /v1/crypto | 10 | 5 | 50 |
| /v1/async | 10 | 5 | 50 |
| /v1/errlogs | 10 | 5 | 50 |
| /v1/jobs | 10 | 5 | 50 |
| /v1/health | 10 | 5 | 50 |
| /v1/cache | 10 | 5 | 50 |
| /v1/cloudaccounts | 10 | 5 | 50 |
| /v1/packs | 10 | 5 | 50 |
| /v1/workspaces | 10 | 5 | 50 |
| /v1/installers | 10 | 5 | 50 |
| /v1/git/webhook | 10 | 5 | 50 |
| /v1/registries | 10 | 5 | 50 |
| /v1/services | 10 | 5 | 50 |
| /v1/overlords | 10 | 5 | 50 |

> | v1/cluster | 10 | 5 | 50 |

Reviewer: ```suggestion
| /v1/cluster | 10 | 5 | 50 |
```
